src/styles/index.scss

```
@@ -1,3 +1,83 @@
-body {
+$day-width: 100px;
+$day-height: 100px;
+$calendar-gap: 1px;
+$padding-prefered: 10px;
+$background-day-color-hover: #ffbfcb;
+$background-day-color: #eee;
+$min-max-day-width: ($day-width * 7 + 2 * $padding-prefered + 6);
+$days: (
+  'mon': 0,
+  'tue': #{$day-width + $calendar-gap},
+  'wed': #{($day-width + $calendar-gap) * 2},
```

```suggestion
  'tue': 1,
  'wed': 2,
```
better to calculate it one time in the margin-left